### DISCUSSION TITLE
Claimed Rewards Never Received in Wallet

### DISCUSSION THREAD

**Discussion by TAvant** [Chia-Network/chia-blockchain#14134] [Support]
When I claim my rewards I never receive them in my wallet.  This has happened twice now. Is anyone else having the same issue? What did you do to resole it? I have tried deleting unconfirmed transactions. Is there a way to see the claimed reward transactions?

**Comment by shaneo257** [comment]
are you talking about the 1.75xch amount, because if you are, its a known issue that this tx doesnt show, but your wallet balance will go up once its confirmed

**Reply by TAvant** [reply]
Thank you for explaining that. What you've described is exactly what is happening.
